pkg/exthttp/transport.go

```diff
@@ -20,6 +20,7 @@ func NewTransport() *http.Transport {
 		}).DialContext,
 		ForceAttemptHTTP2:     true,
 		MaxIdleConns:          100,
+		MaxIdleConnsPerHost:   100,
```

Comment:
If I am not mistaken this one is already covered by the `connectionPoolSizePerHost` option in `cmd/thanos/sidecar.go` that later gets passed down to `NewPrometheusStore()` :P So this should probably be removed.

Reply:
I removed it from here.